jax/_src/config.py

```diff
@@ -1046,6 +1046,16 @@ def _update_jax_memories_thread_local(val):
     ),
 )
 
+share_autotune_config_between_hosts = define_bool_state(
+    name='jax_share_autotune_config_between_hosts',
+    default=False,
+    help=(
+        'If set to True, the coordinator process will share autotune configs '
+        'other participants. This will increase overall compilation time, but '
+        'will lead to equal compiled modules in each process.'
```

Comment:
Can you add something like:
`If this config and jax_share_binary_between_hosts are both set, jax_share_binary_between_hosts will be done when possible and we will fallback to this flag otherwise`.

Otherwise, it isn't clear what happens when both flags are used without reading the code.

Reply:
Done